projects/js-packages/number-formatters/package.json

```
@@ -0,0 +1,38 @@
+{
+	"name": "@automattic/jetpack-number-formatters",
+	"version": "0.1.0",
+	"description": "Number formatting utilities",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/js-packages/number-formatters/#readme",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/js-packages/number-formatters"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Number Formatters"
+	},
+	"dependencies": {
+		"@babel/runtime": "^7"
+	},
+	"devDependencies": {
+		"@babel/core": "7.26.0",
+		"@babel/preset-react": "7.26.3",
+		"@jest/globals": "29.4.3",
+		"@types/jest": "29.5.14",
+		"jest": "29.7.0",
+		"jest-environment-jsdom": "29.7.0",
+		"typescript": "5.0.4"
+	},
+	"type": "module",
+	"exports": {
+		".": "./index.ts"
+	},
+	"scripts": {
+		"build": "pnpm run compile-ts",
```

Continuing on the theme of "setting it up correctly" from https://github.com/Automattic/jetpack/pull/42317#issuecomment-2730416802, you'll likely want to change this something like
```suggestion
		"build": "pnpm run clean && pnpm run compile-ts",
		"clean": "rm -rf build/",
```